rgd/geodata/templates/geodata/rastermetaentry_detail.html

```diff
@@ -55,54 +55,24 @@
 
   <script>
     let extents = JSON.parse('{{ extents|escapejs }}');
-
-    function createImage (src) {
-      let img = new Image();
-      img.crossOrigin = 'anonymous';
-      img.src = src;
-      return img
-    }
-
     var outline = JSON.parse(extents['outline'])["coordinates"][0]
-    // cacheQuads: https://opengeoscience.github.io/geojs/apidocs/geo.quadFeature.html#.spec
-    var quad = layer.createFeature('quad', {cacheQuads: false})
-      .data([
-        {
-          ul: {x: outline[0][0], y: outline[0][1]},
-          ur: {x: outline[1][0], y: outline[1][1]},
-          lr: {x: outline[2][0], y: outline[2][1]},
-          ll: {x: outline[3][0], y: outline[3][1]},
-        },
-      ])
-      .style({
-        color: '#6586c9',
-        opacity: 0.75,
-      })
-      .draw();
+
+    var tiles = map.createLayer('osm');
 
     function updateBandThumbnail () {
       selector = document.getElementById('band-selector')
       var image_id = Number(selector.value);
-      let image = null;
+      console.log(image_id)
       if (image_id >= 0) {
-        var src = extents['thumbnails'][image_id];
-        image = createImage(src);
+        tiles.url(`http://localhost:8000/api/geodata/imagery/image_entry/${image_id}/tiles/{z}/{x}/{y}.jpg`)
```

Comment:
We need to make sure we are not hardcoding the host (e.g. `http://localhost:8000/`) but pulling it from the context. How should I go about this?

Reply:
```js
var host = window.location.protocol + "//" + window.location.host;
```